pkg/api/controller_test.go

```diff
@@ -10835,13 +10835,13 @@ func TestManifestImageIndex(t *testing.T) {
 					resp, err = resty.R().SetHeader("Content-Type", ispec.MediaTypeImageIndex).
 						SetBody(content).Put(baseURL + "/v2/index/manifests/test:1.0")
 					So(err, ShouldBeNil)
-					So(resp.StatusCode(), ShouldEqual, http.StatusBadRequest)
+					So(resp.StatusCode(), ShouldEqual, http.StatusOK)
```

Comment:
See GH test results. This should be 201?

Reply:
fixed